### PR TITLE
Added command: LOAD

### PR DESCRIPTION
**Pull request by skelly37**
# Summary

* This is a…
  * [ ] Bug fix
  * [x] Feature addition
  * [ ] Refactoring
  * [ ] Minor / simple change (like a typo)
  * [ ] Other
* **Describe this change in 1-2 sentences**:
A command that allows to load files as a part of the command pipeline

# Problem

It would be useful to load a file after some commands have already been executed as a part of the pipeline. The `load` command extends positional arguments' functionality.

* JIRA ticket (_optional_): None

# Solution

Added the needed command

# Action
Should be squashed on merge

### CODE REVIEW

**Comment by rdswift** [comment]
Could this be used to load URLs and MBIDs as well?  If so you might be better off passing the argument list to `load_t-Picard()` for processing rather than adding the arguments directly using `add_paths()`.

**Comment by skelly37** [comment]
> Could this be used to load URLs and MBIDs as well? If so you might be better off passing the argument list to `load_to_Picard()` for processing rather than adding the arguments directly using `add_paths()`.

Could be, I wasn't sure if it should be added. What's your take on this @zas @phw ?

**Comment by rdswift** [comment]
> Could be, I wasn't sure if it should be added. What's your take on this @zas @phw ?

My thinking was that it should operate the same as files, etc. entered on the command line, as described by @zas in his comment on https://github.com/metabrainz/picard/pull/2137.

> > So you're saying that we should ditch the positional argument FILE_OR_URL in favor of adding a LOAD command which would be the only way to load something to picard?

> Nope, in addition. Basically picard <FILE> would be equivalent to picard -e LOAD <FILE>. The idea is to let user use only -e commands to achieve the same thing.

**Comment by skelly37** [comment]
Ping @phw @zas , fully done imo

File: picard/tagger.py [hunk 1]
```diff
@@ -480,6 +485,16 @@ def handle_command_fingerprint(self, argstring):
         for album_name in self.albums:
             self.analyze(self.albums[album_name].iterfiles())
 
+    def handle_command_load(self, argstring):
+        if argstring.startswith("command://"):
+            log.error("Cannot LOAD a command: %s", argstring)
+            return
+
+        if not argstring.startswith("file://") and not argstring.startswith("mbid://") and not argstring.startswith("url://"):
+            argstring = "file://" + argstring
```
Comment (zas) [inline picard/tagger.py:494]: This prevents something like `-e load https://musicbrainz.org/release/f1e04a39-65fc-48c9-b9f9-fa1dbe7e9326`.
Instead a check like the one at https://github.com/metabrainz/picard/blob/974547ce32718b35d8da85445887856fa4062fb5/picard/tagger.py#L1335 could be used.

**Review by zas — CHANGES_REQUESTED** [review]

File: picard/tagger.py (hunk 1 above)
Comment (skelly37) [reply picard/tagger.py:494]: My bad, seems like I've gone too automatic... Tho I'd rather keep it consistent with our `ParseItemsToLoad` class, see yourself

Comment (skelly37) [reply picard/tagger.py:494]: Actually, none of it is needed. See the recent commit.

**Review by zas — APPROVED** [review]
LGTM

**Review by phw — APPROVED** [review]

**Pull request merged**